specification/simulate/ingest/SimulateIngestResponseExample3.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Use pipeline substitutions
+description: A successful response when the simulation uses pipeline substitutions.
+# type: response
+# response_code: 200
+value: '{
+  "docs": [
+  {
+  "doc": {
+  "_id": "123",
+  "_index": "my-index",
+  "_version": -3,
+  "_source": {
+  "foo": "foo"
+  },
+  "executed_pipelines": []
+  }
+  },
+  {
+  "doc": {
+  "_id": "456",
+  "_index": "my-index",
+  "_version": -3,
+  "_source": {
+  "bar": "rab"
+  },
+  "executed_pipelines": []
+  }
+  }
+  ]
+  }'
```

Comment:
nit: same on the whitespace here

Reply:
Not sure how this happened, but I've fixed it in 398c37ca28a12e59a1e628433253df59ecb6da23